docs/admin/how_to_release.md

```diff
@@ -1,96 +1,39 @@
----
-layout: longpage
-title: How to Release
-permalink: /dev_docs/admin/how_to_release
----
 
+## Manual Release Testing
```

Comment:
These test descriptions are nice and there are some gems in there (e.g. client should be the first player in a networked game).  Can we document them in `testing.md` and then just link from here?  I wouldn't have thought to look for this level of testing detail under "How to Release."